tube-player/README.md

```diff
@@ -14,7 +14,7 @@ On completion of the workshop, you will have a working cross-platform TubePlayer
 
 ![Completed TubePlayer app](modules/11-App-Finalization/ui-output.gif)
 
-This app is created using Uno Platform and inspired from the work by [Naweed Akram](https://twitter.com/xgeno "@xgeno") on [MAUI](https://github.com/naweed/MauiTubePlayer) and [Flutter](https://github.com/naweed/FlutterTubePlayer)
+These apps was developed using Uno Platform and draws inspiration from [Naweed Akram](https://twitter.com/xgeno "@xgeno")'s [MAUI project](https://github.com/naweed/MauiTubePlayer) and [Flutter project](https://github.com/naweed/FlutterTubePlayer).
```

Comment:
to match simplecalc here 
https://github.com/unoplatform/workshops/blob/9831b2751554fb3a1436999d909a9d3fc8e7ef6d/simple-calc/README.md?plain=1#L8